test/c/eacsl/integer_dividable.t

```diff
@@ -0,0 +1,11 @@
+  $ owi c --e-acsl integer_dividable.c --no-value
+  [kernel] Parsing integer_dividable.c (with preprocessing)
+  [e-acsl] beginning translation.
+  [kernel] Parsing FRAMAC_SHARE/e-acsl/e_acsl.h (with preprocessing)
+  [e-acsl] translation done in project "e-acsl".
```

Comment:
Could you pass a flag to frama-c when asking for the preprocessing file so that it stops writing this kind of verbose output ? (Even better would be to pass the flag only when the `--debug` one is not present in Owi.)

Reply:
now the presence and absence of --debug will set the frama-c verbosity level to 2 and 0, which by default is 1